frontend/src/components/layout/Header.tsx

```diff
@@ -331,7 +333,12 @@ const Header = () => {
               </div>
             </div>
 
-            <Popover.Panel as="nav" className="lg:hidden" aria-label="Global">
+            <Popover.Panel
+              as="nav"
+              className="lg:hidden"
+              aria-label="Global"
+              ref={popoverRef}
```

Comment:
The ref doesn't seem to be needed? Can we remove it?

Reply:
got it, thanks for letting me know! :+1: